altos-rust/altos-core/src/arch/test.rs

```diff
@@ -26,13 +26,16 @@ pub fn yield_cpu() {
     sched::switch_context();
 }
 
-pub fn initialize_stack(stack_ptr: Volatile<usize>, _code: fn(&mut Args), _args: &Box<Args>) -> usize {
+pub fn initialize_stack(stack_ptr: Volatile<usize>, _code: fn(&mut Args), _args: &Box<Args>)
+    -> usize {
+
```

Comment:
I only changed these because the line length was a bit too long. The way I did it here is the same way you modified the functions in free_list.rs when you were doing formatting fixes. I think this way is cleaner as long as the arguments aren't too long by themselves.